### ISSUE TITLE
@angular-contrib/common does not depend on @angular-contrib/core

### ISSUE THREAD

**Issue by pauldraper** [trotyl/angular-contrib#172]
It appears that the npm package @angular-contrib/common does not list @angular-contrib/core as as dependency, but it should.

**Comment by pauldraper** [comment]
Ah, I see that it is a peer dependency. IMO should be a regular dependency; I probably don't need to select both a version of common and core.

**Comment by Yohandah** [comment]
Yeah doing `npm install @angular-contrib/common` as the doc suggests isn't enough, you need to do `npm install @angular-contrib/core` as well. It should be a regular dependency.
